### ISSUE TITLE
Error in Dina Koota

### ISSUE THREAD

**Issue by ykpg** [martin-pe/maitreya8#83]
Error in Dina Koota.
Dina Koota Reported are wrong. Please check.
![Maitreya - Dina Koota](https://github.com/martin-pe/maitreya8/assets/59304682/e0ee8b8a-6960-410d-bf35-8cf1f7590583)

Girl = 1970-July-24, 7:55 PM, Hyderabad, India
Boy = 1960-Aug-02, 8:17 AM, Guntur, India

YK Prakash

**Comment by ossiviljakainen** [comment]
What is the expected koota? Can you please provide screenshot taken from other software which calculates this right? Or can you provide a photo of manual calculation of this? 

Please explain steps how to produce the error and what is the expected calculation result.

**Comment by ykpg** [comment]
Girl = Revati & Boy = Anuradha.
Revati to Anuradha count = 18. Divided by 9 => remainder =  0 taken as 9 = parama Mitra tara.
Anuradha to Revati = 27th - 17th + 1 ( both stars included) = 11 . divided by 9 . remainder = 2. Sampat tara.
website  = https://jyotishy.in

![Image](https://github.com/user-attachments/assets/ffbe1bbb-84e6-425f-9ae8-8552dc8d197e)

**Comment by martin-pe** [comment]
You are right, thank you for reporting. There was an index miscalculation.

The fix will appear in the next release 8.2 (likely in second half of July).
